test/models/batch_invitation_user_test.rb

```diff
@@ -1,6 +1,24 @@
 require "test_helper"
 
 class BatchInvitationUserTest < ActiveSupport::TestCase
+  context ".create!" do
+    setup do
+      @inviting_user = create(:admin_user)
+      @batch_invitation = create(:batch_invitation, :with_organisation, user: @inviting_user)
+    end
+
+    should "strip unwanted whitespace from name before persisting" do
+      user = BatchInvitationUser.create!(
+        name: "  Ailean Millard ",
+        email: "ailean@example.com",
+        batch_invitation: @batch_invitation,
+      )
+
+      assert user.persisted?
```

Comment:
Minor: I think this assertion is redundant, because you've (sensibly) used the bang variation of `BatchInvitationUser.create!`, so an exception would be raised if the user wasn't persisted successfully.

Actually, thinking about it - perhaps it would be worth using FactoryBot's create method here so you don't have to specify the email attribute? I _think_ FactoryBot create uses an ActiveRecord bang method under the hood, so even then you might not need this assertion...